### PR TITLE
Created openproject.conf.sample

### PR DESCRIPTION
**Pull request by HammyHavoc**
This is for https://www.openproject.org/

[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [X] I have read the [contributing](https://github.com/linuxserver/reverse-proxy-confs/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

### CODE REVIEW

File: openproject.conf.sample [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Version 2024/04/13
+# make sure that your OpenProject container is named OpenProject
```
Comment (drizuid) [inline openproject.conf.sample:2]: I believe there are issues on some systems with upper case characters

File: openproject.conf.sample [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    listen [::]:443 ssl;
+
+    server_name openproject.*;
+    include /config/nginx/ssl.conf;
+    client_max_body_size 0;
+
+    location / {    
+        include /config/nginx/proxy.conf;
+        include /config/nginx/resolver.conf;
+        set $upstream_app OpenProject;
```
Comment (drizuid) [inline openproject.conf.sample:16]: I believe there are issues on some systems with upper case characters

**Review by drizuid — COMMENTED** [review]
@nemchik can you confirm?

**Comment by nemchik** [comment]
> @nemchik can you confirm?

Confirmed. Some systems force lowercase container name resolution.

File: openproject.conf.sample [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        set $upstream_port 8080;
+        set $upstream_proto https;
+        proxy_pass $upstream_proto://$upstream_app:$upstream_port;
+        proxy_set_header X-Forwarded-Host $upstream_app:$upstream_port;
```
Comment (nemchik) [inline openproject.conf.sample:20]: This seems wrong. We have `X-Forwarded-Host` set here
https://github.com/linuxserver/docker-swag/blob/41cb7cb10423ac9b2708a55b94e6901955199d49/root/defaults/nginx/proxy.conf.sample#L28
We intentionally use the `$host` variable as it provides information about the visiting client to the header, and thus to the application.

The values you are passing here would essentially be static, and not reflect anything about the client. Is there a documented requirement for this?

**Pull request closed without merging**